index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src="https://kit.fontawesome.com/ea6955352c.js" crossorigin="anonymous"></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="mobile-menu" class="not-active">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#projects">Protfoliu</a></li>
+            <li><a class="mobile-link" href="#aAbout-me">About</a></li>
+            <li>
+              <a class="mobile-link" href="#c#ontact">Contact</a>
```

Comment:
- [ ] You seem to be having a number of typos in your markup which will technically make your menu not functional. There seems to be no ID for "aAbout" and "c#ontact" which will make the page "bookmark" not work. Kindly fix this typo to make your menu links function as expected.